src/app/tests/suites/certification/Test_TC_FAN_2_3.yaml

```
@@ -47,3 +49,5 @@ tests:
       response:
           constraints:
               type: bitmap8
+              minValue: 0
+              maxValue: 4
```

It is possible for a fan to support all motions simultaneously.
```suggestion
              maxValue: 7
```